rails-snow-queen/app/assets/javascripts/quotingPanel.js

```diff
@@ -0,0 +1,49 @@
+class QuotingPanel {
+  constructor() {
+    this.polygons = {};
+    this.totalAreaInSqFt = null;
+    this.totalDue = null;
+  }
+
+  showAddress(geocodedAdress) {
+    if (geocodedAdress) {
+      let addressNode = document.getElementById("displayAddress");
+      while (addressNode.firstChild) { addressNode.removeChild(addressNode.firstChild) }
+      let splitAddress = geocodedAdress.split(",");
+      splitAddress.forEach(element => {
+        let div = document.createElement("div");
+        div.innerText = element;
+        addressNode.appendChild(div);
+      });
+      addressNode.classList.remove("hidden");
+    }
+  }
+
+  updateTotals(polygons) {
+    let SQ_FT_CONVERT = 10.764;
+    const PRICE_PER_SQ_FT = 0.15;
```

Comment:
I would define these in the constructor instead of redefining them every time the method is called